packages/bot-web-ui/src/components/trade-animation/__test__/circular-wrapper.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render } from '@testing-library/react';
+import ContractWrapper from '../circular-wrapper';
+
+describe('ContractWrapper', () => {
+    it('should render <ContractWrapper />', () => {
+        const { container } = render(<ContractWrapper />);
+        expect(container).toBeInTheDocument();
+    });
```

Comment:
@sandeep-deriv  Why is the test case file for ContractWrapper named as circular-wapper.spec.tsx?